tests/contrib/django/test_django_snapshots.py

```diff
@@ -42,15 +40,15 @@ def daphne_client(snapshot):
         env=env,
     )
 
-    client = Client("http://localhost:%d" % SERVER_PORT)
+    client = Client("http://localhost:%s" % port)
 
     # Wait for the server to start up
     client.wait()
 
     try:
         yield client
     finally:
-        resp = client.get_ignored("/shutdown-tracer")
+        resp = client.get_ignored("/shutdown-tracer/")
```

Comment:
why was this change needed?

Reply:
It was added while debugging an error. I will remove it since it doesn't do anything

Reply:
The tests asgi tests that were hitting the shutdown-tracer were failing before this commit: https://github.com/DataDog/dd-trace-py/commit/1f4aaeddb0953f60352a49f131911694810b6218

Reply:
is it because of the route rule being `r"^shutdown-tracer/$"` that implies a required `/` at the end no?

Reply:
also, getting a 500 response seems weird to me, what is the error you get from it?